### PR TITLE
startup-notification: 0.12

### PR DESCRIPTION
**Pull request by aidanharris**
## Description of package

Application startup notification and feedback library

## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).

### CODE REVIEW

**Comment by dylanaraps** [comment]
This package is optional no?

**Comment by aidanharris** [comment]
@dylanaraps Without this i3-gaps fails to build with:

```
configure: error: Package requirements (libstartup-notification-1.0) were not met:

Package 'libstartup-notification-1.0', required by 'virtual:world', not found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables LIBSN_CFLAGS
and LIBSN_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
```

I'll check if there's a feature-flag to turn it off.

**Comment by aidanharris** [comment]
Doesn't look like it's optional. It's listed in the `DEPENDS` file as required.

**Comment by aidanharris** [comment]
List of files that include startup-notification:

```
$ grep -r -E '#include\s.*sn.*\.h' .
./i3-config-wizard/main.c:#include <libsn/sn-launchee.h>
./i3-nagbar/main.c:#include <libsn/sn-launchee.h>
./include/data.h:#include <libsn/sn-launcher.h>
./include/i3.h:#include <libsn/sn-launcher.h>
./include/startup.h:#include <libsn/sn-monitor.h>
./src/handlers.c:#include <libsn/sn-monitor.h>
./src/startup.c:#include <libsn/sn-launcher.h>
./src/util.c:#include <libsn/sn-launcher.h>
./testcases/t/175-startup-notification.t:#include <libsn/sn-common.h>
./testcases/t/175-startup-notification.t:#include <libsn/sn-launchee.h>
```

File: community/startup-notification/depends [hunk 1]
```diff
@@ -0,0 +1,5 @@
+libX11
+libXau
+libxcb
+make
```
Comment (dylanaraps) [inline community/startup-notification/depends]: `make` is an implied dependency.

File: community/startup-notification/build [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr
+
+make PREFIX=/usr
```
Comment (dylanaraps) [inline community/startup-notification/build]: Is `PREFIX` needed here?

File: community/startup-notification/build [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+make DESTDIR="$1" PREFIX=/usr install
```
Comment (dylanaraps) [inline community/startup-notification/build]: Is `PREFIX` needed here?

**Pull request closed without merging**